nttt/tidyup.py

```diff
@@ -1,13 +1,12 @@
+from .constants import Constants
 from .utilities import find_files, find_replace, find_snippet, get_file, save_file
 
-import os
 import os.path
-from pathlib import Path
 
 def fix_meta(src, dst):
     find_replace(src, dst, "  - \n    title:", "  - title:")
 
-def fix_step(src, dst):
+def fix_step(src, dst, lang):
```

Comment:
Let's keep `dst` as the last argument since it's a kind of "output" argument while the other 2 are "input" args

Reply:
Swapped dst and lang.